workflow/rules/read_clipping.smk

```diff
@@ -57,7 +56,7 @@ rule bamclipper:
         " cp {input.bai} {params.output_dir} &&"
         " cd {params.output_dir} &&"
         " bamclipper.sh -b {params.bam} -p {params.bed_path} -n {threads} -u 5 -d 5) "
-        " > {params.cwd}/{log} 2>&1"
+        " > {log} 2>&1"
```

Comment:
What is your value for output_dir? If it it not the cwd, how can this work then (because we do a cd before)?

Reply:
Then the removal of that is only a problem on the local execution of the workflow because on the cluster it will take the absolute path there for `params.cwd` won't be needed, right?

Reply:
Actually this should already be fine, because of the parentheses around the part with the cd. The workdir outside of the parentheses should be unaffected.

Reply:
Ah, my fault. It should be fine AFTER your change. So you were right about the change from the beginning, and my comment was wrong.